topics/atom/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: atom-package, atom-packages, atom-theme
+created_by: GitHub
+display_name: Atom
+github_url: https://github.com/atom
+logo: atom.png
+released: February 26, 2014
+short_description: Atom is a modern open source text editor built with web technologies.
+topic: atom
+url: https://atom.io/
+wikipedia_url: https://en.wikipedia.org/wiki/Atom_(text_editor)
+---
+Atom is a modern open source text editor developed by GitHub. It is designed to be approachable yet customizable with a wide variety of options and packages.
```

Comment:
I like that the short description mentions that Atom is built with web technologies; I thought it would be good to expand on that in the full description. How about -

> Atom is an open-source text editor developed by GitHub. It is designed to be approachable out of the box yet highly customizable. It is built using web technologies so you can customize its look and feel using CSS or add new features using HTML and JavaScript. There are thousands of packages and themes available.

Reply:
I expanded this in 7164960deef93bade7ed17aca66d46916e8d7abf.